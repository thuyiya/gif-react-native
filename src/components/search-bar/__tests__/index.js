/* eslint-disable no-undef */
import React from "react";
// import TestRenderer from "react-test-renderer";
import ShallowRenderer from "react-test-renderer/shallow";
import { SearchBar } from "_components";

describe("SearchBar", () => {
  it("renders correctly react-test-renderer", () => {
    const renderer = new ShallowRenderer();
    renderer.render(
      <SearchBar
        onCancel={() => {}}
        onChangeText={() => {}}
        onChange={() => {}}
      />
    );
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  });
});
