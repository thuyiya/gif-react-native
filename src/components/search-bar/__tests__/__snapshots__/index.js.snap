// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar renders correctly react-test-renderer 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "margin": 16,
    }
  }
>
  <ForwardRef(AnimatedComponentWrapper)
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#f0f0f0",
          "borderRadius": 10,
          "flexDirection": "row",
          "paddingLeft": 8,
          "position": "relative",
        },
        Object {
          "width": 718,
        },
      ]
    }
  >
    <Icon
      name="search"
    />
    <TextInput
      accessibilityHint="Search more Gif"
      accessibilityLabel="Search Input"
      accessible={true}
      allowFontScaling={false}
      clearTextOnFocus={false}
      editable={true}
      maxLength={40}
      onChangeText={[Function]}
      onFocus={[Function]}
      placeholder="Search"
      placeholderTextColor="#919191"
      selectionColor="#333333"
      style={
        Object {
          "fontSize": 16,
          "height": 56,
          "paddingLeft": 8,
          "paddingRight": 8,
          "width": 590,
        }
      }
      value=""
    />
  </ForwardRef(AnimatedComponentWrapper)>
</View>
`;
